rocketpy/simulation/flight.py

```diff
@@ -2019,6 +1961,21 @@ def z(self):
         """Rocket z position as a Function of time."""
         return self.solution_array[:, [0, 3]]
 
+    @property
+    def apogee_x(self):
+        """Rocket x position at apogee."""
+        return self.x.get_value_opt(self.apogee_time)
+
+    @property
+    def apogee_y(self):
+        """Rocket y position at apogee."""
+        return self.y.get_value_opt(self.apogee_time)
+
+    @property
+    def apogee(self):
+        """Rocket z position at apogee."""
+        return self.z.get_value_opt(self.apogee_time)
```

Comment:
I was trying to reduce the number of variables being set during the simulation, in this case, removing the `self.apogee = ...`.

I didn't think about the Monte Carlo loop, but I see you're right, it will probably slow down the whole process.

I will roll this back.

Reply:
Solved: [505275a](https://github.com/RocketPy-Team/RocketPy/pull/581/commits/505275a134f06e5a471b331aa021bcda74024a02)